lib/jellyfish/room_service.ex

```diff
@@ -70,24 +71,32 @@ defmodule Jellyfish.RoomService do
   end
 
   @impl true
-  def handle_call({:create_room, max_peers}, _from, state)
-      when is_nil(max_peers) or (is_integer(max_peers) and max_peers >= 0) do
-    {:ok, room_pid, room_id} = Room.start(max_peers)
-    room = Room.get_state(room_id)
-    Process.monitor(room_pid)
+  def handle_call({:create_room, max_peers, enforce_encoding}, _from, state) do
+    with :ok <- validate_max_peers(max_peers),
+         {:ok, enforce_encoding} <- encoding_to_atom(enforce_encoding),
+         {:ok, room_pid, room_id} <- Room.start(max_peers, enforce_encoding) do
+      room = Room.get_state(room_id)
```

Comment:
Room.start always returns ok, so we don't have to match.
```suggestion
         {:ok, enforce_encoding} <- encoding_to_atom(enforce_encoding) do
      {:ok, room_pid, room_id} = Room.start(max_peers, enforce_encoding)
      room = Room.get_state(room_id)
```